### DISCUSSION TITLE
Custom Loading Layout

### DISCUSSION THREAD

**Discussion by hegoczkigabor** [komarovalexander/ka-table#192] [General]
Hey,

Sorry for the strange question, but is there any way to customize the loading appearance of the table?
https://komarovalexander.github.io/ka-table/#/loading

**Comment by komarovalexander** [comment]
Actually this is a great question. At this time there is no built-in ability to do it. But I will add it in next versions

**Reply by hegoczkigabor** [reply]
Thank you for your answer. Do we have the opportunity to change the default color of the "balls" on the loading indicator? (Without overwriting css classes manually)
